modular/executor/execute_task.go

```diff
@@ -162,108 +163,118 @@ func (e *ExecuteModular) HandleReceivePieceTask(
 		}
 		return
 	}
-	log.CtxDebugw(ctx, "success to handle confirm receive piece task")
+	log.CtxDebugw(ctx, "succeed to handle confirm receive piece task")
 }
 
 func (e *ExecuteModular) HandleGCObjectTask(
 	ctx context.Context,
```

Comment:
These two params can be moved to one line.

Reply:
Updated.